pkgs/applications/misc/plots/default.nix

```diff
@@ -0,0 +1,78 @@
+{ fetchFromGitHub
+, fetchzip
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, python3
+, wrapGAppsHook4
+}:
+let
+  latinmodern-math = fetchzip {
```

Comment:
This font already exist in nixpkgs:

```
$ nix-build -A lmmath && find ./result/share/fonts -type f
/nix/store/72hngp1bqdl5d0m1zp81ynxwlzkdn31c-lmmath-1.959
./result/share/fonts/opentype/latinmodern-math.otf
```

I suggest to use that instead of adding code here to fetch it again.

Reply:
Oh, I just didn't find it because I was looking for `latinmodern`, but it turns out that the package name is `lmmath`.